includes/managers/widgets.php

```diff
@@ -145,6 +150,26 @@ private function init_widgets() {
 		do_action( 'elementor/widgets/register', $this );
 	}
 
+	private function register_widget_by_filename( string $filename ) {
+		$class_name = str_replace( '-', '_', $filename );
+
+		$class_name = __NAMESPACE__ . '\Widget_' . $class_name;
+
+		$this->register( new $class_name() );
+	}
+
+	private function register_atomic_widgets() {
+		$atomic_widgets_filenames = [
+			'atomic-heading',
```

Comment:
shouldn't it be `'a-heading'` as it would have its way into the dom eventually? (or I might be confusing it with something else, not sure)

Reply:
We have decided to name all the 'atomic' related features as atomic-${name}, the only thing that should be shortened is the widget name. That array stores all the widget file names.